sneak-peak.ftd

```diff
@@ -0,0 +1,161 @@
+-- import: fifthtry.github.io/admint.io/assets
+-- import: fifthtry.github.io/admint-cs as cs
+-- import: fifthtry.github.io/admint-typography as typo
+-- import: fifthtry.github.io/admint.io/lib as lp
+
+
+
+
+
+-- ftd.column:
+width: fill-container
+colors: $cs.main
+types: $typo.types
+
+
+
+-- sneak-peak: A sneak peek into the AdMint platform. See how simple it really is!
+image: $assets.files.static.lezgoo.sneak-peak.png
+bg-image: $assets.files.static.lezgoo.sneak-bg.png
+cta-text: Get Started
+cta-url: /
+
+
+-- end: ftd.column
+
+
+
+
+
+
+
+
+
+
+
+
+-- component sneak-peak:
+caption title:
+ftd.image-src image:
+ftd.image-src bg-image:
+integer width: 1440
+string cta-text:
+string cta-url:
+
+-- ftd.column:
+width: fill-container
+align-content: center
+
+-- ftd.column:
+width: fill-container
+align-content: center
+max-width.fixed.px: $sneak-peak.width
+
+-- ftd.column:
+width if { ftd.device != "mobile" }: fill-container
+align-content: center
+margin-horizontal.px if { ftd.device == "mobile" }: 50
+
+-- ftd.image:
+src: $sneak-peak.image
+border-color: $inherited.colors.custom.one
+border-width.px if { ftd.device != "mobile" }: 20
+border-width.px if { ftd.device == "mobile" }: 11
+border-radius.px: 44
+width.fixed.percent: 100
+max-width.fixed.px: 1020
+
+-- end: ftd.column
+
+-- ftd.row:
+if: { ftd.device != "mobile" }
+spacing: space-between
+width: fill-container
+
+-- ftd.column:
+top.percent: 55
+
+-- ftd.text: $sneak-peak.title
+role: $inherited.types.heading-medium
+color: $inherited.colors.text
+width.fixed.px: 750
+margin-left.rem: 5
+anchor: parent
+
+-- end: ftd.column
+
+-- ftd.image:
+src: $sneak-peak.bg-image
+align-self: end
+
+-- end: ftd.row
+
+
+-- ftd.column:
+if: { ftd.device == "mobile" }
+
+-- ftd.image:
+src: $sneak-peak.bg-image
+align-self: end
+width.fixed.percent: 100
+
+-- ftd.column:
+margin-horizontal.px: 12
+margin-vertical.px: 32
+
+-- ftd.text: $sneak-peak.title
+role: $inherited.types.heading-medium
+color: $inherited.colors.text
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+
+
+
+
+
+
+-- ftd.column:
+align-self: start
+margin-left.rem if { ftd.device != "mobile" }: 5
+margin-left.px if { ftd.device == "mobile" }: 12
+
+-- ftd.text: $sneak-peak.cta-text
+background.solid: $inherited.colors.cta-primary.base
+padding-horizontal.px: 41
+padding-vertical.px: 11
+border-radius.px: 50
+role: $inherited.types.button-large
+color: $inherited.colors.cta-primary.text
+link if { sneak-peak.cta-url != NULL }: $sneak-peak.cta-url
+
+-- end: ftd.column
+
+
```

Comment:
@SuryanshShahi there should 1 line space
Also fix it on other places and other files.